### PR TITLE
Improve wording on homepage features

### PR DESCRIPTION
**Pull request by yardenshoham**

### CODE REVIEW

File: docs/src/components/HomepageFeatures.js [hunk 1]
```diff
@@ -8,8 +8,8 @@ const FeatureList = [
     Svg: require('../../static/img/feat-opensource.svg').default,
     description: (
       <>
-        Woodpecker is and will be totally free for ever. As Woodpeckers <a href="https://github.com/woodpecker-ci/woodpecker" target="_blank">source code</a> is
-        OpenSource you can contribute to help evolving the project.
+        Woodpecker is and always be totally free. As Woodpecker's <a href="https://github.com/woodpecker-ci/woodpecker" target="_blank">source code</a> is
```
Comment (6543) [inline docs/src/components/HomepageFeatures.js:11]:
```suggestion
        Woodpecker is and always will be totally free. As Woodpecker's <a href="https://github.com/woodpecker-ci/woodpecker" target="_blank">source code</a> is
```
?

**Comment by 6543** [comment]
is it for hacktoberfest ?

**Comment by yardenshoham** [comment]
> is it for hacktoberfest ?

Just signed up, so I guess yes

**Comment by 6543** [comment]
If you have more time, It would be awesome if you could look bit more throug the docu and if you dont understand or have questions point that out, as this parts need fixing :+1:

**Comment by yardenshoham** [comment]
Sure. By the way, do you want me to copy my latest PR from gitea here?
* go-gitea/gitea#21468

**Comment by 6543** [comment]
uh yes not a bad idea ... just do so

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
@yardenshoham just port the contrib pull to here

**Pull request merged**